src/main/java/ru/itis/controllers/WishListController.java

```diff
@@ -22,29 +22,30 @@ public WishListController(ItemService itemService, WishListService wishListServi
     }
 
     @CrossOrigin
-    @GetMapping("/lists/{list-id}")
+    @GetMapping("/lists/{listId}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Get a single wish list")
-    public WishList getWishList(@PathVariable("list-id") Long listId, @RequestHeader(name = "AUTH") String token) {
-//      TODO: if anonym --> ability to reserve an item
+    public WishList getWishList(@PathVariable Long listId, @RequestHeader(name = "Authorization")
+            String token) {
         WishList defaultWishList = WishList.getDefault();
         return wishListService.findWishListById(listId).orElse(defaultWishList);
     }
 
     @CrossOrigin
-    @PostMapping("/lists/{list-id}")
+    @PostMapping("/lists/{listId}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Add new item to wish list")
-    public void addNewItem(@PathVariable("list-id") Long listId, ItemForm itemForm, @RequestHeader(name = "AUTH") String token) {
-        itemService.addNewItem(itemForm, listId);
+    public void addNewItem(@PathVariable Long listId, ItemDto itemDto, @RequestHeader(name = "Authorization")
+            String token) {
+        itemService.addNewItem(itemDto, listId);
     }
 
     @CrossOrigin
-    @DeleteMapping("/lists/{list-id}")
+    @DeleteMapping("/lists/{listId}")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("Delete an item")
-    public void deleteItem(@PathVariable("list-id") Long listId, @RequestParam("title") String itemName,
-                           @RequestHeader(name = "AUTH") String token) {
+    public void deleteItem(@PathVariable Long listId, @RequestParam("title") String itemName,
```

Comment:
Fix code style. You may use `method(param1, param2, param3)` either
```
method(param1,
       param2,
       param3)
```
Don't mix them.

Reply:
fixed